.github/workflows/ci.yml

```diff
@@ -74,18 +70,18 @@ jobs:
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.9'
+          python-version: '3.10'
 
       - name: Build sdist and wheel
         run: |
           pip install pip setuptools wheel --upgrade
           python setup.py sdist bdist_wheel
 
       - name: Publish a Python distribution to PyPI
-        uses: pypa/gh-action-pypi-publish@v1.5.1
+        uses: pypa/gh-action-pypi-publish@v1.8.10
```

Comment:
better to follow the official recommendation of pinning to `@release/v1`

Reply:
```suggestion
        uses: pypa/gh-action-pypi-publish@v1
```